### DISCUSSION TITLE
2005/08/04/%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9-2

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#43] [Announcements]
# 2005/08/04/%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9-2

最近又得了这种病，不知道怎么搞得

https://blog.qizhen.xyz/2005/08/04/%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9-2

**Comment by ruanqizhen** [comment]
wiser
2005年08月15日 23:17 编辑
我应邀来了! 朋友,说我先是得了过敏性鼻炎,然后是薰麻疹很爽吧! 呵呵,但是很难受Y! 我来过了,改天还来!

回复
戀你の砂
2005年08月9日 12:42 编辑
尽量少吃荤菜，会好的快点~吃清淡一点的~

回复
yoyo
2005年08月4日 20:05 编辑
运动运动左三圈 右三圈 脖子扭扭 屁股扭扭 预备起

回复
